src/app/zeko/sequencer/tests/local_network/gql.ml

```diff
@@ -1505,8 +1449,115 @@ module Queries = struct
         in
         Some cmd_with_hash )
 
+  let pooled_user_commands =
+    field "pooledUserCommands"
+      ~doc:
+        "Retrieve all the scheduled user commands for a specified sender that \
```

Comment:
What is a sender?

Reply:
Good point I've copied that from the original `mina_graphql` code. I've changed that to the `signer` and in regards to zkapp commands to the `fee payer`.